lib/Heartbeat/LED.cpp

```diff
@@ -56,11 +58,13 @@ void LED::setState(bool state) {
   }
 }
 
-bool LED::getState() { return digitalRead(_pin) ^ !_activeHigh; }
+bool LED::getState() {
+  return static_cast<bool>(digitalRead(_pin) ^ !_activeHigh);
```

Comment:
how about `digitalRead(_pin) == _activeHigh`?

Reply:
That would result in the same outcome.

Reply:
but looks much better ;-)

Reply:
done.